### ISSUE TITLE
[Bug] Zoom in / out menu and hotkey options in Linux / KDE do not change font size

### ISSUE THREAD

**Issue by miranda** [lencx/ChatGPT#1252]
### Non-ChatGPT bug

- [X] This issue does not occur on chat.openai.com and only occurs on this app.

### Version

1.1.0

### Bug description

The zoom options do not change font size.  I can't figure out any way to change the font size and it is too small on a 4K monitor.  I have tried both the AppImage and the AUR package.

### OS

Arch Linux, KDE 6, X11

**Comment by wanghao-SunSky** [comment]
MacBook Pro (15-inch, 2016) also encountered this issue.

**Comment by kinimokoena** [comment]
hello i have problem chat gpt is very law can you please assist me

**Comment by kinimokoena** [comment]
and always give wrong information

**Comment by adriandelgg** [comment]
I also can't zoom in and out anymore. It's extremely annoying.

**Comment by adriandelgg** [comment]
@miranda Did you figure it out?

**Comment by miranda** [comment]
> @miranda Did you figure it out?

No... I don't think there is anything to figure out.  Zoom just doesn't work.  Seems like a bug.
I'm stuck using the website until this is fixed because I'm too blind for the font size.

**Comment by adriandelgg** [comment]
> > @miranda Did you figure it out?
> 
> No... I don't think there is anything to figure out. Zoom just doesn't work. Seems like a bug. I'm stuck using the website until this is fixed because I'm too blind for the font size.

Damn, I'm on the same boat..

**Comment by adriandelgg** [comment]
> > @miranda Did you figure it out?
> 
> No... I don't think there is anything to figure out. Zoom just doesn't work. Seems like a bug. I'm stuck using the website until this is fixed because I'm too blind for the font size.

Hey, I actually created a small fix to set the default zoom to any value you want! Here are the instructions:

1. Go to "Control Center"
2. Click on "Scripts"
3. Edit `main.js`
4. Insert this code at the bottom of the `main.js` file (set the zoom amount to whatever value you want. Note that it has to be enclosed in single `''` or double `""` quotes):
```javascript
// zoomAmount must be a string percentage, e.g '150%'
function setDefaultZoom(zoomAmount) {
  const html = document.getElementsByTagName('html')[0];
  html.style.zoom = zoomAmount;
  window.localStorage.setItem('htmlZoom', zoomAmount);
}

setDefaultZoom('150%');
```
5. Hit save and restart the app.

**Comment by miranda** [comment]
> Hey, I actually created a small fix to set the default zoom to any value you want! Here are the instructions:

Awesome, thank you for this, I can finally use this app!

If that's all it takes to change the text size, maybe adding it in the control center would be an easier and better way to fix this bug?

**Comment by adriandelgg** [comment]
> > Hey, I actually created a small fix to set the default zoom to any value you want! Here are the instructions:
> 
> Awesome, thank you for this, I can finally use this app!
> 
> If that's all it takes to change the text size, maybe adding it in the control center would be an easier and better way to fix this bug?

Maybe, that would require more work though. I'm not a maintainer or contributor to this project, I just wanted to get the zoom working again lol

**Comment by tyehuoPHD** [comment]
@adriandelgg  
Thank you bro! Great solution. Finally, i dont need to squeeze to see the characters.

**Comment by sandeshgit** [comment]
Thank you @adriandelgg.

**Comment by Ai80-8** [comment]
Delete it all
